front/src/App.vue

```diff
@@ -1,19 +1,191 @@
+            </el-form-item><el-form-item prop="pass">
+              <el-input class="input"
+                        type="password"
+                        placeholder="Пароль"
+                        v-model="ruleForm.password"
+              >
+              </el-input>
+            </el-form-item>
+          </el-form>
+        </div>
+        <div class="btn_padding">
+          <el-button type="primary" @click="Registration()">Зарегистрироваться</el-button>
+        </div>
+        <div class="entrance">
+          Уже есть аккаунт?
+          <a href="#">Вход</a>
+        </div>
+      </div>
+    </div>
+    <div id="created" class="blackout">
+      <div class="okno">
+        <div class="text">Создать аккаунт
+          <a href="#" class="close">
+            <el-button icon="el-icon-close" circle></el-button>
+          </a>
+        </div>
+        <div class="accountCreate font">
+          <el-button type="success" icon="el-icon-check" circle></el-button>
+          <p>Аккаунт создан!</p>
+          <p>Ты стал частью большого мира, где каждый может внести свою
+            лепту в то, чтобы процесс поиска фильма, сериала или мультфильма был проще и
+            увлекательней.
+          </p>
+          <p>Вперед за просмотрами, оценками и комментариями!</p>
+        </div>
+      </div>
+    </div>
+    <a href="#registration">
+      <el-button type="primary">Зарегистрироваться</el-button>
+    </a>
   </div>
 </template>
 
 <script>
 export default {
   name: 'app',
   data () {
-    return {};
+    return {
+      ruleForm: {
+        userName: '',
+        email: '',
+        password: ''
+      },
+      rules: {
+        email: [
+          { required: true, message: 'Пожалуйста введите email', trigger: 'blur' },
+          { type: 'email', message: 'Пожалуйста введите корректный email', trigger: 'blur' }
+        ],
+        userName: [
+          { required: true, message: 'Введите имя пользователя', trigger: 'blur' }
+        ]
+      }
+    };
+  },
+  methods: {
+    Registration () {
+      fetch('https://localhost:44333/api/user', {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          UserName: this.ruleForm.userName,
+          Email: this.ruleForm.email,
+          UserPassword: this.ruleForm.password
+        })
+      })
+        .then(response => response.text())
+        .then(data => {
+          if (data === '1') {
+            this.ruleForm.userName = this.ruleForm.email = this.ruleForm.password = '';
+            document.location.href = '#created';
+          } else {
+            alert('Данный email уже зарегистрирован');
+          }
+        });
+    }
   }
 };
 </script>
 
-<style></style>
+  <style>
+.blackout {
+  background: rgba(102, 102, 102, 0.5);
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: none;
+}
+.okno {
```

Comment:
is it about window? no russion words, please. You might return to article in wiki about ninja code

Reply:
done